lib/jxl/enc_coeff_order.cc

```diff
@@ -59,32 +59,10 @@ void ComputeCoeffOrder(SpeedTier speed, const ACImage& acs,
                        const FrameDimensions& frame_dim, uint32_t& used_orders,
                        coeff_order_t* JXL_RESTRICT order) {
   std::vector<int32_t> num_zeros(kCoeffOrderMaxSize);
-  // If compressing at high speed and only using 8x8 DCTs, only consider a
-  // subset of blocks.
-  double block_fraction = 1.0f;
-  // TODO(veluca): figure out why sampling blocks if non-8x8s are used makes
-  // encoding significantly less dense.
-  if (speed >= SpeedTier::kSquirrel && used_orders == 1) {
```

Comment:
Shouldn't this be triggered by falcon mode as well as fast JPEG recompression modes?

Reply:
`used_orders == 0` in falcon mode, I think. In any case removing this code doesn't change anything in falcon mode.

Reply:
Ah. How about JPEG recompression>

Reply:
Ah yes, when doing JPEG recompression at efforts 4 to 7, this did trigger.
I'll put it back because in that case (since there is little else to do) it does seem to have some speed impact.